gensim/test/test_utils.py

```diff
@@ -83,6 +90,11 @@ def test_decode_entities(self):
         expected = u'It\x92s the Year of the Horse. YES VIN DIESEL \U0001f64c \U0001f4af'
         self.assertEquals(utils.decode_htmlentities(body), expected)
 
+    def test_check_output(self):
+        if os.name == 'posix':
```

Comment:
What's about windows here?

Reply:
It's about check_output

Reply:
`check_output` should works fine for windows too